### PR TITLE
va-button & va-button-pair: Removing V1 code

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://3014-remove-v1-buttons--65a6e2ed2314f7b8f98609d8.chromatic.com

---

## Description
Closes [#3014](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3014)

## QA Checklist
- [X] Component maintains 1:1 parity with design mocks
- [X] Text is consistent with what's been provided in the mocks
- [X] Component behaves as expected across breakpoints
- [X] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [X] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [X] Tab order and focus state work as expected
- [X] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [X] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [X] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
N/A

## Acceptance criteria
- [X] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-button/va-button.scss [hunk 1]
```diff
@@ -43,73 +43,4 @@
...
-  font-size: 1.5rem;
-  padding: 0.9375rem 1.875rem;
-}
-
-:host([uswds='false'][disabled]:not([disabled='false'])) button {
-  background-color: var(--vads-color-base-lighter);
-  box-shadow: none;
-  color: var(--vads-button-color-text-primary-alt-on-light);
-}
+@import '../../mixins/focusable.css';
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-button/va-button.scss:46]: If this import is still necessary, can it be added to the top of the stylesheet with the other imports?

**Review by jamigibbs — COMMENTED** [review]
The Chromatic link is giving a 404 for me.

**Comment by Andrew565** [comment]
@jamigibbs Ready for re-review.

**Review by powellkerry — APPROVED** [review]
This should have a major label because it is removing functionality. Other than that it looks good to me.

File: packages/web-components/src/components/va-omb-info/va-omb-info.css [hunk 2]
```diff
@@ -2,7 +2,3 @@
   display: block;
   font-size: 1.06rem;
 }
-
-strong {
-  font-weight: 700;
```
[the remark this answers is not in the document]
Comment (Andrew565) [reply packages/web-components/src/components/va-omb-info/va-omb-info.css:7]: This particular style is redundant. By default `strong` tags are rendered by browsers as though they were `font-weight: 700;`. To be sure I removed this style and tested it in my browser, and there was no difference. So I went ahead and removed it.

**Comment by pmclaren19** [comment]
@Andrew565 I think this pr introduced a bug that made it so the v1 version of the `va-file-input` no longer exists/works....

**Pull request merged**